dspace-api/src/main/java/org/dspace/xmlworkflow/storedcomponents/PoolTask.java

```diff
@@ -35,7 +40,10 @@
  */
 @Entity
 @Table(name = "cwf_pooltask")
-public class PoolTask implements ReloadableEntity<Integer> {
+public class PoolTask implements ReloadableEntity<Integer>, BrowsableDSpaceObject<Integer> {
+
+    @Transient
+    public transient Map<String, Object> extraInfo = new HashMap<String, Object>();
```

Comment:
@abollini : Again, I agree with @KevinVdV here. This looks like it can be cleaned up now, as you removed `extraInfo` from the `BrowsableObject` interface.

Reply:
done